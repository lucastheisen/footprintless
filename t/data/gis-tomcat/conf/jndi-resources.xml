<Environment name="cas/serverName" 
  type="java.lang.String" 
  override="false"
  value="https://asias-portal-dev-viz.mitre.org" />
<Environment name="cas/AuthenticationFilter/casServerLoginUrl" 
  type="java.lang.String" 
  override="false"
  value="https://asias-portal-dev-sso.mitre.org/portal-cas/login" />
<Environment name="cas/ArcGisCas30TicketValidationFilter/casServerUrlPrefix" 
  type="java.lang.String" 
  override="false"
  value="https://asias-portal-dev-sso.mitre.org/portal-cas/" />
