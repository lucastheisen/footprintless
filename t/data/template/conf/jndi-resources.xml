<Environment name="foo/principalName" 
  type="java.lang.String" 
  override="false"
  value="foo" />
<Environment name="foo/allowedFrom" 
  type="java.lang.String" 
  override="false"
  value="localhost,${T{trusted.hostname}}${T_perl{$internal = property('trusted.internal_hostname');$internal?",$internal":''}}" />
