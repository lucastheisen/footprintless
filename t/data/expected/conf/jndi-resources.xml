<Environment name="foo/principalName" 
  type="java.lang.String" 
  override="false"
  value="foo" />
<Environment name="foo/allowedFrom" 
  type="java.lang.String" 
  override="false"
  value="localhost,trusted.pastdev.com,trusted.pastdev.local" />
